clients/js/.eslintignore

```diff
@@ -1 +1,2 @@
 protos
+examples
```

Comment:
Any reason to keep them out of eslint and prettier

Reply:
mostly because it's test code

Reply:
Even tests are under formatter and lint if I am not wrong. Would be preferrable to have everything under the formatter.

Reply:
ack

Reply:
done